facebook-commerce.php

```diff
@@ -1398,6 +1416,41 @@ function update_product_group( $woo_product ) {
 		*/
 	}
 
+
+	/**
+	 * Determine if there matching variation to the default attributes
+	 *
+	 * @param WC_Facebook_Product $woo_product
+	 *
+	 * @return WC_Product_Variation|null
+	 */
+	protected function get_product_group_default_variation( $woo_product ) {
```

Comment:
Yes, because in runtime the product data type passed to the parent calling method `WC_Facebookcommerce_Integration::update_product_group` is by the same data type of `\WC_Facebook_Product` ... so to keep the workflow the same without breaking it, I just continued with the same data type